### PR TITLE
Add Dialpad to Calling and CallWithChat sample apps

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

Update application home pages to have dial pad and create a different selection to do 1:N calling
# Why

Allows us to demonstrate how to use the Dialpad component to start calls with our composites. 

Splitting the screens allows us to drive simplicity to starting a call so there is less confusion on what information you need to provide to start what call

https://skype.visualstudio.com/SPOOL/_workitems/edit/2920799
# How Tested

Ran locally on the sample apps to verify dialpad works. will test sample apps with 1:N POC to verify ACS calling.

### CODE REVIEW

File: samples/Calling/src/app/views/HomeScreen.tsx [hunk 1]
```diff
@@ -145,15 +180,29 @@ export const HomeScreen = (props: HomeScreenProps): JSX.Element => {
...
+
+/* @conditional-compile-remove(PSTN-calls) */
+/**
+ * Parse the unformatted string that is given from the dialpad to something that the
+ * Calling adapter will accept as a PSTN participant.
+ *
+ * @param participant - string stored from dialpad inputs.
+ * @returns formatted phone number to be added to call locator.
+ */
+const parseDialPadParticipant = (participant?: string): string[] | undefined => {
```
Comment (anjulgarg) [inline samples/Calling/src/app/views/HomeScreen.tsx:203]: This should ideally be happening in the dialpad itself.
I can't think of a scenario where a formatted string returned by the dialpad is useful.

**Review by carocao-msft — APPROVED** [review]

File: samples/Calling/src/app/styles/HomeScreen.styles.ts [hunk 2]
```diff
@@ -59,6 +59,17 @@ export const buttonStyle = mergeStyles({
...
   padding: '0.625rem'
 });
+/* @conditional-compile-remove(PSTN-calls) */
 export const outboundtextField = mergeStyles({
   paddingTop: '0.5rem'
 });
+/* @conditional-compile-remove(PSTN-calls) */
+export const addDialparticipantStyles = mergeStyles({
+  fontWeight: 600,
+  fontSize: '0.875rem', // 14px
```
[the remark this answers is not in the document]
Comment (anjulgarg) [reply samples/Calling/src/app/styles/HomeScreen.styles.ts:69]: Not sure why we need to provide font size here?
Isn't the default font size working?

File: samples/Calling/src/app/views/HomeScreen.tsx [hunk 3]
```diff
@@ -100,20 +119,40 @@ export const HomeScreen = (props: HomeScreenProps): JSX.Element => {
...
+                      placeholder={"Comma seperated ACS user ID's"}
+                      onChange={(_, newValue) => setOutboundParticipants(newValue)}
+                    />
+                  </Stack>
+                </Stack>
+              )
+            }
+            {
+              /* @conditional-compile-remove(PSTN-calls) */ pstnCallChosen && (
+                <Stack>
```
Comment (anjulgarg) [inline samples/Calling/src/app/views/HomeScreen.tsx:137]: Why the 2 stacks? Seem redundant

**Review by anjulgarg — APPROVED** [review]

File: samples/Calling/src/app/views/HomeScreen.tsx (hunk 3 above)
Comment (dmceachernmsft) [reply samples/Calling/src/app/views/HomeScreen.tsx:137]: Hold over from previous version, will remove

**Pull request merged**